tests/util/linkComponents.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env mocha */
```

Comment:
I was going to say it might be reasonable to add this env as an [override](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns) for the `tests/util/*.js` glob, but apparently that feature is only available in ESLint v4.1.0+, and we have `"^3.0.0 || ^4.0.0 || ^5.0.0"` in our devDependencies.
Would it perhaps make sense to put an `.eslintrc` in `tests/util` to use [cascading](https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) instead?

Reply:
`.eslintrc` added to `tests/util`